qaframework-bdd-tests/src/test/resources/features/refapp-2.x/015-reports/reports.feature

```diff
@@ -14,4 +14,4 @@ Feature: Reports Management
     And user enters start date
     And user enters end date
     And user clicks on request report button
-    Then user clicks on view report link
+    #Then user clicks on view report link
```

Comment:
@Muta-Jonathan is this change part of the ticket requirement?

Reply:
@kdaud according to https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/qaframework-bdd-tests/src/test/resources/features/refapp-2.x/015-reports/reports.feature ...its commented out on master....and my PR is failing to update as on master

Reply:
This makes the change being reflected as part of the ticket requirement which false.

>  my PR is failing to update as on master

do you find this link helpful? https://gist.github.com/whoisryosuke/36b3b41e738394170b9a7c230665e6b9